_source/logzio_collections/_log-sources/cloudwatch.md

```diff
@@ -75,7 +75,7 @@ In the _Environment variables_ section, set your Logz.io account token, URL, and
 |---|---|
 | TOKEN <span class="required-param"></span> | {% include log-shipping/replace-vars.html token='noReplace' %} <!-- logzio-inject:account-token --> |
 | URL <span class="required-param"></span> | Protocol, listener host, and port (for example, `https://<<LISTENER-HOST>>:8071`). <br > {% include log-shipping/replace-vars.html listener=true %} <!-- logzio-inject:listener-url --> |
-| TYPE <span class="required-param"></span> | The log type you'll use with this Lambda. This can be a [built-in log type]({{site.baseurl}}/user-guide/log-shipping/built-in-log-types.html), or a custom log type. <br> You should create a new Lambda for each log type you use. |
+| TYPE <span class="default-param">`logzio_cloudwatch_lambda`</span> | The log type you'll use with this Lambda. This can be a [built-in log type]({{site.baseurl}}/user-guide/log-shipping/built-in-log-types.html), or a custom log type. <br> You should create a new Lambda for each log type you use. |
 | FORMAT <span class="default-param">`text`</span> | `json` or `text`. If `json`, the Lambda function will attempt to parse the message field as JSON and populate the event data with the parsed fields. |
 | COMPRESS <span class="default-param">`false`</span> | Set to `true` to compress logs before sending them. Set to `false` to send uncompressed logs. |
```

Comment:
We're missing `ENRICH`: https://github.com/logzio/logzio_aws_serverless/tree/master/python2/cloudwatch

Reply:
Is this also true for the cloudformation deployment?

Reply:
Indeed, great catch. @ronish31 can you please add it to the cloudformation template

Reply:
yes

Reply:
@idohalevi I'm told `TYPE` is not required